test/gtest/ucp/ucp_test.cc

```diff
@@ -844,6 +845,11 @@ ucs_status_t ucp_test_base::entity::listen(listen_cb_type_t cb_type,
         params.conn_handler.cb    = reject_conn_cb;
         params.conn_handler.arg   = reinterpret_cast<void*>(this);
         break;
+    case LISTEN_CB_CUSTOM:
+        params.field_mask        |= UCP_LISTENER_PARAM_FIELD_CONN_HANDLER;
+        params.conn_handler.cb    = custom_cb->cb;
```

Comment:
pass only CB itself, assume arg is "this"

Reply:
"this" here is an instance of entity class. I need an instance of `test_ucp_sockaddr_conn_request` class,to get a listener in custom callback on which to reject.
```
test_ucp_sockaddr_conn_request *self =
            reinterpret_cast<test_ucp_sockaddr_conn_request*>(arg);
        ucp_listener_reject(self->receiver().listenerh(), conn_request);
```